python/tests/sanity_checks/test_sanity_check.py

```diff
@@ -0,0 +1,51 @@
+import os
+import pandas as pd
+from sanity_checks.sanity_check import check_pct_values
+
+# Current working directory
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+TEST_DIR = os.path.join(THIS_DIR, os.pardir, "data", "sanity_checks")
+
+CDC_RESTRICTED = {
+    'age_county': os.path.join(TEST_DIR, 'cdc_restricted_age_county.json'),
+    'sex_county_time': os.path.join(TEST_DIR, 'cdc_restricted_sex_county_time.json'),
+    'sex_county': os.path.join(TEST_DIR, 'cdc_restricted_sex_county.json'),
+    'sex_national': os.path.join(TEST_DIR, 'cdc_restricted_sex_national.json'),
+    'sex_state': os.path.join(TEST_DIR, 'cdc_restricted_sex_state.json')
+}
+
```

Comment:
I think in this file you should also test an error case. Like a case where you expect it to fail and error. See https://github.com/SatcherInstitute/health-equity-tracker/blob/89f6457cf5f1a4dcef520cfd349ce73a0de61025/python/tests/ingestion/test_dataset_utils.py#L278-L281 as an example of how to do that.